Makefile

```diff
@@ -27,16 +27,24 @@ clean: ## Removes build and test artifacts
 	@find . -name '*~' -exec rm -f {} +
 	@find . -name '__pycache__' -exec rm -rf {} +
 
-.PHONY: docker-build
-docker-build:
-	@echo "==> Building docker image for local testing"
-	@docker build -t dag_factory:latest .
+
+.PHONY: copy-files
+copy-files: ## copy project in docker context
+	mkdir -p dev/include/tmp_dagfactory
+	cp -r ./dagfactory dev/tmp_dagfactory
+	cp ./pyproject.toml dev/include/tmp_dagfactory/pyproject.toml
+	cp ./LICENSE dev/include/tmp_dagfactory/LICENSE
+	cp ./README.md dev/include/tmp_dagfactory/README.md
```

Comment:
Why are we copying all these files? Would there be an alternative (e.g. to mount `dagfactory` in a folder that is part of the PYTHONPATH in the `Dockerfile`? Or some other..?

Reply:
> Why are we copying all these files? Would

We need these files to install dagfactory local changes in astro project.

> Would there be an alternative (e.g. to mount dagfactory in a folder that is part of the PYTHONPATH in the Dockerfile? Or some other..?

I attempted using a symbolic link, but it didn’t work. I can look into a mounting approach, though I'm not sure at the moment if it will work or not.

Reply:
An alternative could be to build a wheel, and install using it - this way we'll avoid copying all the files